lua/nvim-tree/colors.lua

```diff
@@ -33,6 +33,7 @@ local function get_hl_groups()
   return {
     IndentMarker = { fg = "#8094b4" },
     Symlink = { gui = "bold", fg = colors.cyan },
+    FolderSymlink = { gui = "bold", fg = colors.orange },
```

Comment:
Unfortunately this is a breaking change: all users will see this new highlight where there was none previously.

We cannot set this group by default, however we can default it to `FolderName` to preserve current behaviour.